modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeDefinitionLocalServiceImpl.java

```diff
@@ -102,6 +105,8 @@ public ListTypeDefinition deleteListTypeDefinition(
 			throw new RequiredListTypeDefinitionException();
 		}
 
+		_validateInvokerBundle(listTypeDefinition.isSystem());
```

Comment:
I think this validation should come before RequiredListTypeDefinitionException(), this way we avoid making a count query, does this makes sense?